package.json

```diff
@@ -7,12 +7,15 @@
   "homepage": "https://firebase.google.com/",
   "scripts": {
     "build": "gulp build",
-    "lint": "tslint --project tsconfig-lint.json --format stylish",
+    "lint": "run-s lint:src lint:unit lint:integration",
```

Comment:
`npm-run-all` supports glob-like pattern matching, so you can do `run-s lint:*`.

Why did you decide to run these in sequence instead of parallel?